ts/src/bybit.ts

```diff
@@ -8245,7 +8245,7 @@ export default class bybit extends Exchange {
         return tiers;
     }
 
-    parseMarketLeverageTiers (info, market: Market = undefined): LeverageTier[] {
+    parseMarketLeverageTiers (info: Dict, market: Market = undefined): LeverageTier[] {
```

Comment:
this is not right 
![Uploading image.png…]()

Reply:
![image](https://github.com/ccxt/ccxt/assets/43336371/c617cf2d-6a93-422f-899d-b1ffb2652771)